lib/sendf.h

```diff
@@ -360,7 +360,7 @@ curl_off_t Curl_creader_client_length(struct Curl_easy *data);
  * Ask the installed reader at phase CURL_CR_CLIENT to start
  * reading from the given offset. On success, this will reduce
  * the `total_length()` by the amount.
- * @param date    the transfer to read client bytes for
+ * @param data    the transfer to read client bytes for
  * param offset   the offset where to start reads from, negative
```

Comment:
This second line should be `@param` too, right?

Reply:
Thanks. changed.